crane_x7_examples_py/setup.py

```diff
@@ -0,0 +1,37 @@
+from glob import glob
+import os
+
+from setuptools import find_packages, setup
+
+package_name = 'crane_x7_examples_py'
+
+setup(
+    name=package_name,
+    version='0.1.0',
+    packages=find_packages(exclude=['test']),
+    data_files=[
+        ('share/ament_index/resource_index/packages', ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+        (os.path.join('share', package_name,
+                      'launch'), glob(os.path.join('launch', '*launch.[pxy][yma]*'))),
+        (os.path.join('share', package_name, 'config'), glob(os.path.join('config', '*'))),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='RT Corporation',
+    maintainer_email='shop@rt-net.jp',
+    description='python examples of Sciurus17 ROS package',
```

Comment:
```suggestion
    description='python examples of CRANE-X7 ROS package',
```
`Sciurus17`のままになっているので`CRANE-X7`に修正するとよいかと思います。